docs/codeReview.md

```diff
@@ -35,3 +35,15 @@ A PR advances from self-review to reviewer-review and finally maintainer review
     - reply if you didn't
 - If you need to clarify parts of the code, check if it can be done by adding comments or improve naming of variables/functions/classes
 - When you replied or resolved all comments, move issue back to reviewer- or maintainer-review
+
+## Experiment
+
+We want to run an experiment for two months. During the experiment, all PRs will still require a Reviewer Review and a Maintainer Review. However, if the developer creating the PR feels that only one review is required, they can add a label: “One Review Required”. The Reviewer Reviewer then adds a comment saying whether they agree that the PR could be merged once they’ve approved it.
+
+Examples of PRs that could be safe to merge that way
+
+- Minor changes, e.g. [remove octokit](https://github.com/ParabolInc/parabol/pull/6479)
+- Small, non-architectural changes that happen behind a feature flag, e.g. [show GitLab issues in the Estimate phase](https://github.com/ParabolInc/parabol/pull/6355)
+- Work that is related to new features which are hidden after a feature flag and do not touch existing functionality (any work related to the architecture of the new feature is an exception and should go through the full review process) e.g [added end team prompt mutation](https://github.com/ParabolInc/parabol/pull/6250)
```

Comment:
This feels like something that should need maintainer review _eventually_, since the feature flag will (hopefully) eventually be removed when the feature moves to GA.

No change needed for the experiment since maintainers will be reviewing these PRs later on anyway, but I think any future governance should address this in some way (e.g. "maintainer review not required to merge, but required before turning on the feature flag for more than X customers")

Reply:
It'd be hard to do the reviews after the code is already merged but what we can do is keep track of the PRs and manual tests that are needed to pass before removing the feature flag. The real intention is to move fast with things that don't require maintainer review in the first place ie. I'd still want to improve the way we split the work, submit smaller PRs, ask for feedback early and discuss all the architecture-related changes with the maintainer, preferably before doing the work. But yeah, let's see how it goes and learn as we go.